src/store/metadata/stub/content/editions/versions/adult-obesity-prevalence-aged-18-years-and-over_2023-05.json

```diff
@@ -0,0 +1,125 @@
+{
+    "@context": "https://staging.idpd.uk/ns#",
+    "@id": "https://staging.idpd.uk/datasets/adult-obesity-prevalence-aged-18-years-and-over/editions/2023-05/versions",
+    "@type": "hydra:Collection",
+    "title": "Adult obesity prevalence (aged 18 years and over)",
+    "versions": [
+        {
+            "@type": [
+              "dcat:Distribution",
+              "csvw:Table"
+            ],
+            "@id": "https://staging.idpd.uk/datasets/adult-obesity-prevalence-aged-18-years-and-over/editions/2023-05/versions/1",
+            "identifier": "1",
+            "issued": "2023-05-17T12:41:00Z",
+            "title": "Adult obesity prevalence (aged 18 years and over)",
+            "summary": "Percentage of adults aged 18 and over classified as living with obesity, England, 16/11/2015-15/11/2016 to 16/11/2021-15/11/2022.",
+            "description": "Percentage of adults aged 18 and over classified as living with obesity, in England, for 16/11/2015-15/11/2016 to 16/11/2021-15/11/2022.\n\nCoverage\nEngland\n\nGeography definition\nLocal Authority Districts and Unitary Authorities,\nCounties,\nRegions,\nCombined Authorities,\nNation (England)\n\nSource\nEngland: [Office for Health Improvement and Disparities]\n\nSource Notes and Caveats\nAdults are defined as obese if their body mass index (BMI) is greater than or equal to 30kg/m²",
```

Comment:
Please implement all the changes requested for the editions file on the versions file as well.